### PR TITLE
pending rule action of admin

### PR DESCRIPTION
**Pull request by nguyenngocthuan98**
borrow requests could been accept, deny, delete or make returned book by admin

### CODE REVIEW

**Comment by nguyenngocthuan98** [comment]
ready

File: app/Http/Controllers/Admin/BorrowController.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    public function pay($id)
+    {
+        $status = Borrow::where('id', '=', e($id))->first();
+        if ($status) {
+            $status->accept = 3;
+            $status->save();
+
+            return redirect()->back();
+        }
+    }
```
Comment (quoctd-0607) [inline app/Http/Controllers/Admin/BorrowController.php:53]: 3 hàm trên giống nhau, em nên gộp lại 1 hàm.

**Comment by nguyenngocthuan98** [comment]
ready

File: app/Http/Controllers/Admin/BorrowController.php (hunk 1 above)
Comment (nguyenngocthuan98) [reply app/Http/Controllers/Admin/BorrowController.php:50]: dạ

**Comment by quoctd-0607** [comment]
commeted
conflict code rồi e

**Comment by quoctd-0607** [comment]
commented

**Comment by nguyenngocthuan98** [comment]
ready

**Comment by nguyenngocthuan98** [comment]
ready

**Pull request merged**